source/onboard/user-provisioning-workflows.rst

```diff
@@ -11,7 +11,7 @@ User Provisioning
 There are currently three recommended user provisioning workflows in Mattermost:
 
 1. **On demand:** If user accounts are not pre-provisioned using one of the methods described below, then a new user account will be provisioned when the user first logs in. When the user logs in, they are asked to select a public team to join (all users must belong to at least one team) and then they are added automatically to Town Square and Off-Topic channels. Mattermost also has a default channel setting that allows everyone to be added to additional channels specified by the organization.
-2. **Pre-provisioned via bulk import:** Mattermost features a `bulk data loading tool <https://docs.mattermost.com/deployment/bulk-loading.html>`__ that can be used for pre-provisioning new users by adding them to teams and channels before their first login to Mattermost. This tool automates the creation of Teams, Channels, Users and Posts (with file attachments). It can also be used to migrate users and content from an existing system.
+2. **Pre-provisioned via bulk import:** Mattermost features a `bulk data loading tool <https://docs.mattermost.com/onboard/bulk-loading-data.html>`__ that can be used for pre-provisioning new users by adding them to teams and channels before their first login to Mattermost. This tool automates the creation of Teams, Channels, Users and Posts (with file attachments). It can also be used to migrate users and content from an existing system.
```

Comment:
```suggestion
2. **Pre-provisioned via bulk import:** Mattermost features a `bulk data loading tool <https://docs.mattermost.com/onboard/bulk-loading-data.html>`__ that can be used for pre-provisioning new users by adding them to teams and channels before their first login to Mattermost. This tool automates the creation of Teams, Channels, Users, and Posts (with file attachments). It can also be used to migrate users and content from an existing system.
```